### PR TITLE
MGMT-6413 Add current deployment state dashboard

### PR DESCRIPTION
**Pull request by oshercc**
image

### CODE REVIEW

**Comment by oshercc** [comment]
@nmagnezi

File: dashboards/grafana-dashboard-assisted-installer-deployment-overview.configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+            ],
+            "fields": "",
+            "values": false
+          },
+          "textMode": "value_and_name"
+        },
+        "pluginVersion": "7.2.1",
+        "targets": [
+          {
+            "expr": "increase(sum(service_assisted_installer_cluster_creations{clusterId=~\".+\", openshiftVersion=~\"$OpenShift_Version\"})[$__range:$__rate_interval])",
```
Comment (nmagnezi) [inline dashboards/grafana-dashboard-assisted-installer-deployment-overview.configmap.yaml:89]: why does this specific one have both `expr` and `rawSql`?

**Review by nmagnezi — COMMENTED** [review]
Looks good. Just one question inline.

Proposal (not a review, just in case it is interesting to you): 
with this, you could probably count SNO clusters and non-SNO.
for the non-SNO, you could probably calculate avg hosts count per cluster.

File: dashboards/grafana-dashboard-assisted-installer-deployment-overview.configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+              }
+            ]
+          },
+          {
+            "format": "table",
+            "group": [ ],
+            "hide": false,
+            "metricColumn": "none",
+            "rawQuery": true,
+            "rawSql": "select count(*) AS \"Deregisterd clusters pre delete\" from clusters where deleted_at IS not NULL  ",
```
Comment (rwsu) [inline dashboards/grafana-dashboard-assisted-installer-deployment-overview.configmap.yaml:124]: s/Deregisterd/Deregistered

**Review by rwsu — COMMENTED** [review]
Small nit.

**Review by nmagnezi — APPROVED** [review]
/lgtm

**Pull request merged**